portfolio/src/main/webapp/script.js

```diff
@@ -39,8 +161,7 @@ function createCommentElement(comment) {
   deleteButtonElement.addEventListener("click", () => {
     deleteComment(comment);
 
-    // Remove the comment from the DOM.
-    commentElement.remove();
+    commentElement.remove(); // Remove the comment from the DOM.
   });
 
```

Comment:
Also in L181 `document.getElementById("comment-list").remove();` do you really want to remove that element container? `loadComments()` references this container element.